h2/src/main/org/h2/engine/Mode.java

```diff
@@ -23,7 +23,25 @@
      */
     static final String REGULAR = "REGULAR";
 
-    private static final HashMap<String, Mode> MODES = New.hashMap();
+    public static enum DbTypeEnum {
```

Comment:
you're going to need to merge to master, someone else already added a mode enum :-)